lib/index.js

```diff
@@ -1,2 +1,2 @@
-module.exports = require('./components/Modal');
-
+import Modal from './components/Modal.js';
+export default Modal;
```

Comment:
I think it would be better(?) to use `export-default-from`, but this is stage-1.
What do you think?

`export-default-from`

```js
export default from './components/Modal';
```